src/components v2/Forms/Atoms/Label/Label.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <div class="photon-label">
+    <span class="label">{{ label }}</span>
+    <span v-if="isRequired" class="required-astrix">*</span>
```

Comment:
misspelled `asterisks`

Reply:
I am sorry sir. English is my second language 🤣